tests/features/camel-jcache/src/main/java/org/apache/karaf/camel/test/CamelJcacheHzRouteSupplier.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.karaf.camel.test;
+
+import org.apache.camel.CamelContext;
+import org.apache.karaf.camel.itests.CamelRouteSupplier;
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+        name = "karaf-camel-jcache-hz-test",
+        immediate = true,
+        service = CamelRouteSupplier.class
+)
+public class CamelJcacheHzRouteSupplier extends AbstractCamelJcacheRouteSupplier {
+
+    public static final String PROVIDER_CLASS = "com.hazelcast.cache.HazelcastCachingProvider";
+
+    @Override
+    public void configure(CamelContext camelContext) {
+        System.setProperty("hazelcast.jcache.provider.type", "member");
```

Comment:
Use `ClusterProperty.JCACHE_PROVIDER_TYPE.setSystemProperty("member")` instead

Reply:
it has a side effect : importing the `com.hazelcast.spi.properties` causes the ehcache test to fail because it doesn't install the camel-hazelcast feature. so I changed the dependency to optional in maven to make it work